src/components/AnalysisModal.js

```diff
@@ -148,33 +149,34 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
     const galaxyApp = currentApp(tools.galaxy.label)
     const cromwellApp = currentApp(tools.cromwell.label)
 
+    // TODO: Try to move app/tool-specific info into tools (in notebook-utils.js) so the function below can just iterate over tools instead of duplicating logic
     const renderToolButtons = () => div({
       style: { display: 'flex', alignItems: 'center', flexDirection: 'column', justifyContent: 'space-between' }
     }, [
-      div({
+      h(Clickable, {
         style: styles.toolCard, onClick: () => {
           setCurrentTool(tools.Jupyter.label)
           enterNextViewMode(tools.Jupyter.label)
         }
-      }, [img({ src: jupyterLogoLong, style: _.merge(styles.image, { width: '30%' }) })]),
-      div({
+      }, [img({ src: jupyterLogoLong, alt: 'Create new notebook', style: _.merge(styles.image, { width: '30%' }) })]),
+      h(Clickable, {
         style: styles.toolCard, onClick: () => {
           setCurrentTool(tools.RStudio.label)
           enterNextViewMode(tools.RStudio.label)
         }
-      }, [img({ src: rstudioBioLogo, style: styles.image })]),
-      div({
+      }, [img({ src: rstudioBioLogo, alt: 'Create new R markdown file', style: styles.image })]),
+      h(Clickable, {
         style: { opacity: galaxyApp ? '0.5' : '1', ...styles.toolCard }, onClick: () => {
           setCurrentTool(tools.galaxy.label)
           enterNextViewMode(tools.galaxy.label)
         }, disabled: !galaxyApp, title: galaxyApp ? 'You already have a galaxy environment' : ''
-      }, [img({ src: galaxyLogo, style: _.merge(styles.image, { width: '30%' }) })]),
+      }, [img({ src: galaxyLogo, alt: 'Create new Galaxy app', style: _.merge(styles.image, { width: '30%' }) })]),
       !tools.cromwell.isAppHidden && div({
```

Comment:
Please make this a Clickable too. And then I think you can link [IA-3043](https://broadworkbench.atlassian.net/browse/IA-3043) to this PR as well.

Reply:
Ugh…I don't know why I keep missing the Cromwell one.
Just pushed a commit and linked to the other ticket.